src/test/java/uk/nhs/tis/trainee/notifications/event/ProgrammeMembershipListenerIntegrationTest.java

```diff
@@ -647,36 +667,93 @@ void shouldSendMinimalNotificationsWhenTemplateVariablesMissing(NotificationType
     assertThat("Unexpected content.", content.html(), is(expectedContentStr));
   }
 
+  @ParameterizedTest
+  @EnumSource(value = NotificationType.class, names = "PROGRAMME_POG_MONTH_12")
+  void shouldSendFullPogNotificationsWhenTemplateVariablesPresent(NotificationType type)
```

Comment:
There didn't seem much point in other tests for 'minimal' emails to demonstrate the missing data rules, but possibly I was just getting a bit tired.

Reply:
Given the volume of code changes, and the ricketiness of the existing codebase, we could do 150% and I'd bet we'd still find a bug 😈 . I do bet that.